app/src/main/res/layout/bottom_sheet_add_fav.xml

```diff
@@ -22,27 +22,33 @@
                 android:layout_height="wrap_content"
                 android:layout_gravity="top"
                 />
-            <androidx.coordinatorlayout.widget.CoordinatorLayout
+            <!-- SearchBar at the top -->
+            <com.google.android.material.search.SearchBar
+                android:id="@+id/search_bar_bus"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent">
-                <com.google.android.material.search.SearchBar
-                    android:id="@+id/search_bar_bus"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    android:hint="@string/hint_fav_search"
-                    />
-
-                <com.google.android.material.search.SearchView
-                    android:id="@+id/search_view_bus"
-                    android:layout_width="match_parent"
-                    android:layout_height="wrap_content"
-                    app:layout_anchor="@id/search_bar_bus"/>
-            </androidx.coordinatorlayout.widget.CoordinatorLayout>
-
-            <TextView
+                android:layout_height="wrap_content"
+                android:hint="Type in a stop code!"
+                app:layout_scrollFlags="scroll|enterAlways" />
+
+            <androidx.recyclerview.widget.RecyclerView
+                android:id="@+id/recycler_view_suggestions"
+                android:layout_width="match_parent"
+                android:layout_height="wrap_content"
+                app:layout_behavior="androidx.coordinatorlayout.widget.CoordinatorLayout$LayoutParams" />
+
+            <com.google.android.material.search.SearchView
+                android:id="@+id/search_view_bus"
                 android:layout_width="match_parent"
-                android:layout_height="match_parent"
-                android:text="Maybe a list of suggested bus stops can go here, like recently searched busses or nearby" />
+                android:layout_height="wrap_content"
+                app:layout_anchor="@id/search_bar_bus">
+            </com.google.android.material.search.SearchView>
```

Comment:
The search view has a specific implementation that makes things a little less janky, see https://github.com/material-components/material-components-android/blob/master/docs/components/Search.md#search-history-suggestions-and-results

In general the steps are
1. Put the search bar and search view inside a Coordinator layout
2. Put the recycler view within the search view

That way the UI won't look as janky (sorry that's on me for implementing it that way initially)

Reply:
👍